src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -1,24 +1,25 @@
 package core.basesyntax.service.impl;
 
-import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoTransaction;
+import core.basesyntax.model.FruitTransaction;
 import core.basesyntax.service.ReportService;
 import java.util.List;
 
 public class ReportServiceImpl implements ReportService {
```

Comment:
this service shouldn't work with transactions. As an input, it could take your data from the storage. Consider that this service is only responsible for generation of reports after all other processings (of transactions) are already done and persisted

Reply:
not fixed - report should work state of the storage - how many fruits of which type we have - and then write this into to a file in some specific format